force_wfmanager/left_side_pane/new_entity_modal.py

```diff
@@ -153,3 +190,126 @@ def get_plugin_from_factory(self, factory):
             plugin = factory.mco_factory.plugin
 
         return plugin
+
+    def _get_current_model_editable(self):
+        """A check which indicates if 1. A view with at least
+        one item exists for this model and 2. That those items
+        are actually visible to the user"""
+        if self.current_model is None:
+            return False
+
+        view_info = self.view_structure()
+
+        # Remove any non-visible traits
+        view_info = [trait_name for trait_name in
```

Comment:
mostly stylistic as we normally split long list comprehensions. we put the for and the if on different lines as the first word, so it reads more or less like English.